ldclient.go

```diff
@@ -327,6 +330,14 @@ func MakeCustomClient(sdkKey string, config Config, waitFor time.Duration) (*LDC
 	if waitFor > 0 && client.dataSource != datasource.NewNullDataSource() {
 		loggers.Infof("Waiting up to %d milliseconds for LaunchDarkly client to start...",
 			waitFor/time.Millisecond)
+
+		// If you use a long duration and wait for the timeout, then any network delays will cause
+		// your application to wait a long time before continuing execution.
+		if waitFor.Seconds() > HighWaitForSeconds {
+			loggers.Warnf("Make Client was called with a timeout greater than %d. "+
```

Comment:
```suggestion
			loggers.Warnf("Client was created was with a timeout greater than %d. "+
```
So the wording works for MakeClient and MakeCustomClient